enclave-runner/src/usercalls/mod.rs

```diff
@@ -336,59 +343,292 @@ impl EnclaveState {
         })
     }
 
+    fn create_worker_threads(scope : &crossbeam::thread::Scope ,num_of_threads: usize,
+                             work_receiver: crossbeam::crossbeam_channel::Receiver<Work>,
+                             io_queue_send: mpsc::Sender<UsercallSendData>,
+                             enclave: Arc<EnclaveState>) {
+        for _ in 1..num_of_threads {
+            let work_receiver = work_receiver.clone();
+            let io_queue_send = io_queue_send.clone();
+            let enclave = enclave.clone();
+
+            scope.spawn(move |_| {
```

Comment:
Why are you using a crossbeam scope for thread spawning? Using a `move` closure pretty much defeats the purpose of that.